From 303c80d69eb72a78d0bb7908d2038cbc3493f3af Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sat, 11 May 2019 11:18:35 +0200
Subject: [PATCH] meson: allow building a static library

So that passing "-Ddefault_library=both" also creates a static lib.

Note that Libs.private in the .pc file will still be wrong because of
https://github.com/mesonbuild/meson/issues/3934 (it contains things like
-lpixman-mmx)
---
 pixman/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pixman/meson.build b/pixman/meson.build
index 6ce87e7..7b66827 100644
--- a/pixman/meson.build
+++ b/pixman/meson.build
@@ -97,7 +97,7 @@ pixman_files = files(
   'pixman-utils.c',
 )
 
-libpixman = shared_library(
+libpixman = library(
   'pixman-1',
   [pixman_files, config_h, version_h],
   link_with : [pixman_simd_libs],
-- 
2.21.0

